app/javascript/components/venue/VenueShow.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Axios from 'axios';
+
+export default class VenueShow extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { venue: {} };
+    this.handleDelete = this.handleDelete.bind(this);
+  }
+
+  componentDidMount() {
+    Axios.get(`/api/v1/venues/${this.props.match.params.id}`)
+      .then(res => {
+        const venue= res.data;
+        this.setState({venue});
+      })
+  }
+
+  handleDelete() {
+    Axios.delete(`/api/v1/venues/${this.props.match.params.id}`)
+      .then(() => {
+        this.props.history.push("/venues")
+      })
+  }
+
+  render() {
+    return (
+      <div className="jumbotron">
+        <h2>{this.state.venue.address}</h2>
+        <p> <strong> ID:</strong> {this.state.venue.id}</p>
+        <p><strong> Description:</strong>{this.state.venue.description}</p>
+        <p>
+          <Link to={`/venues/${this.state.venue.id}/edit`} className="btn btn-outline-dark">Edit</Link> 
```

Comment:
it's better to move such url to const

Reply:
Maybe, but for me, as a new player with JS, it's better actually see what is going on here without abstraction

Reply:
I think const isn't a problem

Reply:
Explain, why do I need them here? Why it is so necessary for you? And take a look at yours views, please.